public/app/plugins/panel/canvas/editor/element/elementEditor.tsx

```diff
@@ -36,10 +36,10 @@ export function getElementEditor(opts: CanvasEditorOptions): NestedPanelOptions<
 
     // Note that canvas editor writes things to the scene!
     values: (parent: NestedValueAccess) => ({
-      getValue: (path: string) => {
+      getValue: (path) => {
         return lodashGet(opts.element.options, path);
       },
-      onChange: (path: string, value: any) => {
+      onChange: (path, value) => {
```

Comment:
In these cases, is it that it implicitly already has a non-`any`, but it has been overridden by `any`?

Reply:
implicitly it's still `any` (you can see the definition for `onChange` [here](https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/utils/OptionsUIBuilders.ts#L165)), so this doesn't really improve the typing so much as decrease the overall count of `any`s by 1.

**but** i still think that's important. it's very hard to be motivated to fix these type errors properly when there's 2000 remaining. once we get down to like 100 then there's light at the end of the tunnel.